client/tests/integration/smartcontracts/executor_custom_data_model/src/parameters.rs

```diff
@@ -0,0 +1,23 @@
+//! Module with custom parameters
```

Comment:
Looks like `parameters.rs` and `permissions.rs` are not used anywhere, at least there are no mod declarations for them in `lib.rs`

Reply:
my bad, committed by accident